src/objects/InstancedMesh.js

```diff
@@ -21,15 +28,64 @@ InstancedMesh.prototype = Object.assign( Object.create( Mesh.prototype ), {
 
 	isInstancedMesh: true,
 
-	raycast: function () {},
+	getMatrixAt: function ( index, matrix ) {
```

Comment:
We need to add this method to the TS declaration file and the docs. The TS declaration should be:
```ts
getMatrixAt( index: number, matrix: Matrix4 ): void
```

Reply:
OK, let me finish it